src/charts/charts.less

```diff
@@ -11,6 +11,32 @@ pf-c3-chart {
     width: 100%;
   }
 }
+.pct-donut-chart-pf {
+
+  .pct-donut-chart-pf-block {
+    .pct-donut-chart-pf-label {
+      display: block;
+    }
+  }
+
+  .pct-donut-chart-pf-left,
+  .pct-donut-chart-pf-right {
+
+    pf-c3-chart {
+      display: inline-block;
+    }
+
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .pct-donut-chart-pf-left {
+    flex-direction: row-reverse;
+  }
+}
+
```

Comment:
After looking a little bit more on this, looks like we have a choice. The appearance is, it looks like we attempted to start a PF core aspect to the charts, here:
- https://github.com/cdcabrera/patternfly/blob/master/src/less/charts.less
- https://rawgit.com/patternfly/patternfly/master-dist/dist/tests/donut-charts.html

With a distinction about only how we modify SVG aspect of the Charts within AngularPF. However it looks like we slightly diverged a little and started placing a few more bits inside AngularPF instead of core. 

I can go back and move this newer piece over to core now, or hold off based on what's already in AngularPf? But in addition, outside of this PR, we probably need to go through the rest of charts.less and separate them out a bit more.